### ISSUE TITLE
Producers need to be adapted for FSM

### ISSUE THREAD

**Issue by arummler** [eudaq/eudaq#277]
Producers in the eudaq repo:
- [ ] TimepixProducer  
- [ ] altroUSB  
- [ ] cmspixel  
- [ ] eudrb     
- [ ] fortis    
- [ ] mimosa32  
- [ ] mvd  
- [ ] palpidefs  
- [ ] pixelmanproducer  
- [ ] rpi-controller  
- [ ] timepixdummy
- [ ] altro            
- [ ] calice    
- [ ] depfet    
- [ ] explorer  
- [ ] mimoroma  
- [ ] miniTLU   
- [ ] ni   
- [ ] palpidess  
- [ ] root             
- [ ] taki            
- [X] tlu

Producers outside the eudaq repo
- [ ] STcontrol
- [ ] RCEofflineproducer
- [ ] Pysub

**Comment by dreyling** [comment]
The ExampleProducer (in main) is also updated. This can be used as an example.

**Comment by arummler** [comment]
Deactivated all producers not adapted yet for CI: #279 Please reactivate whenever a producer works again.

**Comment by arummler** [comment]
Do will still need eudrb? Does the hardware exist anymore?

**Comment by dreyling** [comment]
I don't think so. It is the old read out hardware, which is lying in our museum.
Page 8 of http://cds.cern.ch/record/2000979/files/AIDA-SLIDE-2015-039.pdf

**Comment by arummler** [comment]
That's what I thought, then I suppose we can remove the producer.

**Comment by arummler** [comment]
calice: @airqui

**Comment by arummler** [comment]
cmspixel: @simonspa

**Comment by arummler** [comment]
explorer: @freidt

**Comment by arummler** [comment]
palpidess: @freidt

**Comment by arummler** [comment]
palpidefs: @freidt ,  jacobus van hoorne

**Comment by arummler** [comment]
minitlu: @eyiliu

**Comment by arummler** [comment]
root: @eyiliu

**Comment by arummler** [comment]
depfet: @hperrey who now?

**Comment by freidt** [comment]
explorer, palpidess, mimosa32: obsolete -> removed ( #285 )
palpidefs: adapted ( #285 )

**Comment by airqui** [comment]
I have adapted and the Calice producers for the new FSM (dummy application).
I also have tested the new feature and works for us without any problem. 

As a minor comment, I would propose to separate the buttons in the gui in two sets: left for init and config and right for start,stop, terminate for example. There are two many buttons together...

**Comment by eyiliu** [comment]
Producer  minitlu requestg cactus/uhal which is provied as binary only for SLC. I recommand to disable the building in CI.

**Comment by PatrickAhl** [comment]
pyBAR SiLab-Bonn/pyBAR#62 solved for EUDAQ 1.7 and higher

**Comment by laborleben** [comment]
Can someone tick the pyBAR checkbox?

**Comment by laborleben** [comment]
Please use pyBAR https://github.com/SiLab-Bonn/pyBAR/tree/v3.1.2 or 
https://github.com/SiLab-Bonn/pyBAR/tree/v2.2.2 or higher.

Tested in November 2018 during DEPFET test beam at DESY.
